bin/evaluate-measurements.js

```diff
@@ -98,22 +98,22 @@ async function processRound (roundIndex, measurements, resultCounts) {
   })
 
   for (const m of round.measurements) {
-    if (m.fraudAssessment !== 'OK') continue
+    if (m.taskingEvaluation !== 'OK') continue
     resultCounts.total++
     resultCounts[m.retrievalResult] = (resultCounts[m.retrievalResult] ?? 0) + 1
   }
 
   if (!keepRejected) {
     round.measurements = round.measurements
       // Keep accepted measurements only
-      .filter(m => m.fraudAssessment === 'OK')
+      .filter(m => m.taskingEvaluation === 'OK')
```

Comment:
This is fixing the bug I wanted to fix in https://github.com/filecoin-station/spark-evaluate/pull/396.

Are we okay with that, or should I preserve the current behaviour since refactorings are expected to not change observable behaviour?

Reply:
See https://github.com/filecoin-station/spark-evaluate/issues/446